testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvMultinodeAllConnectors.java

```diff
@@ -62,6 +62,7 @@ public void extendEnvironment(Environment.Builder builder)
                         "kinesis",
                         "kudu",
                         "localfile",
+                        "mariadb",
```

Comment:
Did you need to remember to add this or did some test catch this? (Unrelated to review, asking generally).

Reply:
I think we need to remember this class and there is no test to catch it.